src/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivative.hpp

```diff
@@ -375,19 +378,32 @@ ComputeTimeDerivative<Metavariables>::apply(
           domain::Tags::DetInvJacobian<Frame::Logical, Frame::Inertial>>(box);
     }
   }
-
-  detail::volume_terms<system, volume_dim,
-                       compute_volume_time_derivative_terms>(
-      make_not_null(&box), make_not_null(&volume_fluxes),
-      make_not_null(&partial_derivs), make_not_null(&temporaries),
-      db::get<variables_tag>(box), db::get<::dg::Tags::Formulation>(box), mesh,
-      db::get<domain::Tags::Coordinates<volume_dim, Frame::Inertial>>(box),
-      db::get<::domain::Tags::InverseJacobian<volume_dim, Frame::Logical,
-                                              Frame::Inertial>>(box),
-      det_inverse_jacobian,
-      db::get<::domain::Tags::MeshVelocity<volume_dim>>(box),
-      db::get<::domain::Tags::DivMeshVelocity>(box),
-      typename compute_volume_time_derivative_terms::argument_tags{});
+  db::mutate_apply<
+      tmpl::list<dt_variables_tag>,
+      typename compute_volume_time_derivative_terms::argument_tags>(
+      [&dg_formulation, &det_inverse_jacobian,
+       &div_mesh_velocity = db::get<::domain::Tags::DivMeshVelocity>(box),
+       &evolved_variables = db::get<variables_tag>(box),
+       &inertial_coordinates =
+           db::get<domain::Tags::Coordinates<volume_dim, Frame::Inertial>>(box),
+       &logical_to_inertial_inv_jacobian =
+           db::get<::domain::Tags::InverseJacobian<volume_dim, Frame::Logical,
+                                                   Frame::Inertial>>(box),
+       &mesh,
+       &mesh_velocity = db::get<::domain::Tags::MeshVelocity<volume_dim>>(box),
```

Comment:
Because then I have to deal with that `mutate_apply` forces an argument ordering on me, and that doesn't work with the pointer parts, or I have to reorder all the arguments so that maybe it works. I also can't split a parameter pack, which makes that switch impossible in this case.

Reply:
Okay, fair enough